### PR TITLE
Add signposts in git commit messages

### PR DESCRIPTION
**Pull request by mattwynne**
### 🤔 What's changed?

* Added a body to the commit messages, with a link back to this repo
* Modified the test runner slightly to capture the body as well as the subject of a commit message.

### ⚡️ What's your motivation? 

Communication: Make it more likely for people to discover this tool and perhaps use it, or hack on it and improve it.

### 🏷️ What kind of change is this?

- :zap: New feature (non-breaking change which adds new behaviour)

### 📋 Checklist:

- [x] I agree to respect and uphold the [Cucumber Community Code of Conduct](https://cucumber.io/conduct/)
- [x] I've changed the behaviour of the code
  - [x] I have added/updated tests to cover my changes.

----

*This text was originally generated from a [template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates), then edited by hand. [You can modify the template here.](https://github.com/cucumber/.github/edit/main/.github/PULL_REQUEST_TEMPLATE.md)*

### CODE REVIEW

**Review by mpkorstanje — COMMENTED** [review]
This feels like belts and braces to me. It's already documented in the releasing guide how to make releases. 

It also seems like the wrong sign post because it skips the releasing guide.

**Comment by mattwynne** [comment]
I just have a bit of an aversion to things being done by bots and scripts that look like they were done by a human. I'd prefer to have some kind of notice attached to the commits to make it clear that they weren't done by hand.

I can add a link to the releasing guide too - would that make you happier?

**Comment by mpkorstanje** [comment]
I'd prefer not using links all together. They go stale.

Maybe something like:

```
[polyglot-release] Prepare release v1.0.0
```
I've seen the maven plugin do that. I have no idea if there is any other sort of convention.

https://github.com/cucumber/cucumber-jvm/commit/37d3cd9e717ac63b6c0988f120fdbee9cd6c8bc6

Or maybe that's too much faff in the title. So

```
Prepare release v1.0.0

Made by polyglot-release v1.2.3
```

The version number would be nice for debugging if we stop automatic updates of the script.

(Unrelated, we have the same update problem with the changelog tool, maybe it needs to print it's version too somewhere).

**Comment by mattwynne** [comment]
Makes sense.

**Comment by mpkorstanje** [comment]
I'll have a look at versioning this tool then. We're getting close to stable.

File: polyglot-release [hunk 1]
```diff
@@ -325,6 +325,12 @@ function log() {
   fi
 }
 
+function format_commit_message() {
+  echo "$1
+
+Created-by: polyglot-release v${POLYGLOT_RELEASE_VERSION:--unknown}"
```
Comment (mpkorstanje) [inline polyglot-release:331]: Used a syntax that looks like the co-author syntax. For easy grepping.

**Review by mpkorstanje — APPROVED** [review]

**Comment by mpkorstanje** [comment]
This MR now depends on #65 because I needed the version number for the commit message. I've set the target branch to [add-up-to-date-check](https://github.com/cucumber/polyglot-release/tree/add-up-to-date-check) so the diff remains readable. But we'll have to merge with master after #65 is merged.

**Pull request merged**